_topic_map.yml

```diff
@@ -1638,6 +1638,8 @@ Topics:
     File: adding-bare-metal-compute-user-infra
 - Name: Deploying machine health checks
   File: deploying-machine-health-checks
+- Name: Poison Pill Operator for remidiating unhealthy nodes
```

Comment:
A question more than a comment. 
I've noticed 'Poison Pill Operator' is fully spelt out every time, is it ever / or can it be abbreviated to PPO?

Reply:
@ogradyp We can't abbreviate it- has been spelled out in all engineering documents and PRs.